### ISSUE TITLE
airpods pro 2 requires two connect commands to connect 

### ISSUE THREAD

**Issue by alishtamimi** [Acidham/alfred-airpod-connector#19]
I have a weird issue where I have to type the connect command twice to get my Airpods connected, the first one doesn't do anything, while the second one always works, this happens when my Airpods are connected to the iPhone, and I try to move them back to the mac.

**Comment by Acidham** [comment]
Yes, I am aware of that, and it is related to the auto-switching feature from Apple. You can disable if you wish: https://www.cnet.com/tech/services-and-software/airpods-max-pro-automatic-switching-in-ios-14-is-annoying-heres-how-to-turn-it-off/
